events/message.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
         } else {
             const guildDbInfo = await client.db.table('guildConfig').get(msg.guild.id).run();
             if (guildDbInfo['official']) return require('../util/points.js').run(client, msg.author, 2);
-            if (guildDbInfo['premium']) return require('../util/points.js').run(client, msg.author, 2);
+            else if (guildDbInfo['premium']) return require('../util/points.js').run(client, msg.author, 2);

```

Comment:
You sure that's right syntax?

Reply:
What's wrong with it?

Reply:
I don't know, I've just never seen it before haha, that change isn't really necessary as in both of the if statements there is a return statement so the next one won't be fired anyway.

Reply:
I'll merge if its valid syntax though

Reply:
Generally two "if"s are used if both events can be fired, it just provides better visibility

Reply:
Else if is always been valid javascript syntax

Reply:
I know that - but without curly braces? Never seen that before. Will merge though.

Reply:
Yea one liners are valid without curly braces, in everything from "for", to "else if", to "else" statements. etc.